client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -351,7 +351,7 @@ class CalypsoifyIframe extends Component<
 				this.props.siteCreationFlow === 'gutenboarding' && this.props.isSiteUnlaunched;
 			const frankenflowUrl = `${ window.location.origin }/start/new-launch?siteSlug=${ this.props.siteSlug }&source=editor`;
 			const isNewLaunch = config.isEnabled( 'gutenboarding/new-launch' );
-			const isExperimental = new URLSearchParams( window?.location.search ).has( 'latest' );
+			const isExperimental = config.isEnabled( 'gutenboarding/feature-picker' );
```

Comment:
This "isExperimental" = "feature-picker" can feel a bit confusing later on but I think it's fine now, and we'll sort it out later.

Reply:
Right now this would be easiest way to flip the switch between multi-column PlansGrid and accordion PlansGrid + Feature picker for both Gutenboarding and Site Setup flows.

Reply:
Makes sense 👍